db/migrate/20200504085933_add_schedule_manually_to_work_packages.rb

```diff
@@ -0,0 +1,5 @@
+class AddScheduleManuallyToWorkPackages < ActiveRecord::Migration[6.0]
+  def change
+    add_column :work_packages, :schedule_manually, :boolean, default: false
```

Comment:
We might want to add an index on this as as we will differentiate by that attribute when scheduling. But we can also add this later on when actually required.